### PR TITLE
game: add Flappy Gopher game to catalog

### PR DESCRIPTION
**Pull request by deadprogram**
This PR is to add "Flappy Gopher" to the Firefly Zero catalog.

### CODE REVIEW

File: authors/deadprogram.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name:     deadprogram
+pronouns: he/him
+links:
+  github:   https://github.com/deadprogram/
+  homepage: https://deadprogrammersociety.com/
+short: Having some fun with games on Firefly Zero using TinyGo.
+about: >
+  Technologist for hire, open source developer, businessperson, author, speaker, iconoclast.
+  TinyGo team member - https://tinygo.org
+  Ringleader of The Hybrid Group - hybridgroup.com
```
Comment (orsinium) [inline authors/deadprogram.yaml:10]: Since you use `>`, it's all rendered as a single line 

image

Comment (orsinium) [reply authors/deadprogram.yaml:10]: You can also use markdown in here to make proper clickable links.

Comment (deadprogram) [reply authors/deadprogram.yaml]: See updates.

Comment (orsinium) [reply authors/deadprogram.yaml]: Still not pretty. 

image

Please, try rendering it locally:

```
task
firefox ./public/deadprogram.html 
```

Comment (deadprogram) [reply authors/deadprogram.yaml]: I do not have `task` installed, and many other people will probably not either?

Comment (deadprogram) [reply authors/deadprogram.yaml]: also, that taskfile is installing some python things without setting up any venv first, which is usually not too good of an idea.

Comment (deadprogram) [reply authors/deadprogram.yaml]: I removed all that from the .yml file to simplify this process for myself.

Comment (deadprogram) [reply authors/deadprogram.yaml]: To clarify, not the taskfile.yml, from the .yml file I added in this PR with my info.

Comment (orsinium) [reply authors/deadprogram.yaml]: Thank you! Feel free to send another PR if you ever want to update your intro :)

**Pull request merged**